ss58-registry.json

```diff
@@ -460,11 +460,20 @@
       "standardAccount": "*25519",
       "website": "https://composable.finance"
     },
+    {
+      "prefix": 51,
+      "network": "turing",
+      "displayName": "Turing Network",
+      "symbols": ["TUR"],
+      "decimals": [10, 10],
+      "standardAccount": "*25519",
+      "website": "https://oak.tech/turing/home/"
```

Comment:
Let’s use this one for website, "https://oak.tech/turing/token/"

Reply:
Fixed